CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # CHANGELOG
 
+## `v12.1.0-beta`
```

Comment:
Should new API versions be included?

Reply:
We talked and we prefer not to include every new api version (as you can see in the v12 release), just to anounce the new services.

Reply:
but created an issue if we want to do that, to do it automatically https://github.com/Azure/azure-sdk-for-go/issues/931

Reply:
Good :) I guess if we included new API versions it would be overkill